scripts/merge-coverage.ts

```diff
@@ -0,0 +1,23 @@
+import { readdirSync, renameSync, rmSync } from 'fs';
+import { join } from 'path';
+
+const coverageDir = join(__dirname, '../coverage/');
+const environmentsDir = join(coverageDir, '/environments/');
+const validEnvironments = ['browser', 'node'];
+
+const main = () => {
+  const environments = readdirSync(environmentsDir);
+  environments.forEach(async (environment) => {
```

Comment:
If we stop `await`ing inside the inner method.

```suggestion
  environments.forEach((environment) => {
```